src/sentry/mediators/external_requests/alert_rule_action_requester.py

```diff
@@ -57,16 +73,11 @@ def _make_request(self) -> Mapping[str, Union[bool, str]]:
                     "error_message": str(e),
                 },
             )
-            # NOTE: bool(e.response) is always False because non-2xx responses are Falsey.
-            text = e.response.text if e.response is not None else None
-            message = f"{self.sentry_app.name}: {text or DEFAULT_ERROR_MESSAGE}"
+            message = f"{self.sentry_app.name} Error: {self._get_error_message(e.response)}"
             # Bubble up error message from Sentry App to the UI for the user.
-            return {"success": False, "message": message}
-
-        body_raw = safe_urlread(request)
-        body = body_raw.decode() if body_raw else None
-        message = f"{self.sentry_app.name}: {body or DEFAULT_SUCCESS_MESSAGE}"
```

Comment:
@NisanthanNanthakumar  I don't believe we were using the message anywhere for successful alert rules created, so I removed any code that looked at the response, and just opted to always use the default success message.  Am I understanding it correctly?

Reply:
bumping this question @NisanthanNanthakumar

Reply:
@leeandher yea, we don't currently show the success response text from the SentryApp in the UI. However, that may change in the future. I think it's best to surface the success response text in the API response. IMO, it doesn't really matter if the UI currently considers it to be unnecessary.

Reply:
Sure, added and wrote some tests for it as well.